exercises/protein-translation/canonical-data.json

```diff
@@ -0,0 +1,143 @@
+          "property": "translate",
+          "strand": "UUC",
+          "expected": "Phenylalanine"
+        },
+        {
+          "description": "Leucine RNA sequence 1 is identified",
+          "property": "translate",
+          "strand": "UUA",
+          "expected": "Leucine"
+        },
+        {
+          "description": "Leucine RNA sequence 2 is identified",
+          "property": "translate",
+          "strand": "UUG",
+          "expected": "Leucine"
+        },
+        {
+          "description": "Serine RNA sequence 1 is identified",
+          "property": "translate",
+          "strand": "UCU",
+          "expected": "Serine"
+        },
+        {
+          "description": "Serine RNA sequence 2 is identified",
+          "property": "translate",
+          "strand": "UCC",
+          "expected": "Serine"
+        },
+        {
+          "description": "Serine RNA sequence 3 is identified",
+          "property": "translate",
+          "strand": "UCA",
+          "expected": "Serine"
+        },
+        {
+          "description": "Serine RNA sequence 4 is identified",
+          "property": "translate",
+          "strand": "UCG",
+          "expected": "Serine"
+        },
+        {
+          "description": "Tyrosine RNA sequence 1 is identified",
+          "property": "translate",
+          "strand": "UAU",
+          "expected": "Tyrosine"
+        },
+        {
+          "description": "Tyrosine RNA sequence 2 is identified",
+          "property": "translate",
+          "strand": "UAC",
+          "expected": "Tyrosine"
+        },
+        {
+          "description": "Cysteine RNA sequence 1 is identified",
+          "property": "translate",
+          "strand": "UGU",
+          "expected": "Cysteine"
+        },
+        {
+          "description": "Cysteine RNA sequence 2 is identified",
+          "property": "translate",
+          "strand": "UGC",
+          "expected": "Cysteine"
+        },
+        {
+          "description": "Tryptophan RNA sequence is identified",
+          "property": "translate",
+          "strand": "UGG",
+          "expected": "Tryptophan"
+        },
+        {
+          "description": "STOP codon RNA sequence is identified",
+          "property": "translate",
+          "strand": "UAA",
+          "expected": "STOP"
+        },
+        {
+          "description": "STOP codon RNA sequence is identified",
+          "property": "translate",
+          "strand": "UAG",
+          "expected": "STOP"
+        },
+        {
+          "description": "STOP codon RNA sequence is identified",
+          "property": "translate",
+          "strand": "UGA",
+          "expected": "STOP"
+        },
+        {
+          "description": "Translate RNA strand into correct protein list",
+          "property": "translate",
+          "strand": "AUGUUUUGG",
+          "expected": ["Methionine","Phenylalanine","Tryptophan"]
```

Comment:
The type returned by `translate` should be consistent; until this line, the output of `translate` has been a string, but now a list is expected. The python test cases use two different properties `of_codon` and `of_rna` for these; something similar would be appropriate.

Reply:
This feels a lot like what has been discussed recently at #983.  "translate_codon" seems to be an intermediate test of what the description is really asking for.  `"The translate_codon"` may not belong.  @petertseng, @Insti thoughts?

Reply:
@rpottsoh I believe @ErikSchierboom beat you to it. See his comments.